lib/controllers/migrate-controller.ts

```diff
@@ -1182,6 +1192,41 @@ export class MigrateController
 		return dependencies;
 	}
 
+	private async migrateTSConfig(tsConfigPath: string): Promise<boolean> {
+		try {
+			const configContents = this.$fs.readJson(tsConfigPath);
+
+			// update
+			configContents.compilerOptions = configContents.compilerOptions || {};
+			configContents.compilerOptions.target = "es2017";
+			configContents.compilerOptions.module = "esnext";
+			configContents.compilerOptions.moduleResolution = "node";
+			configContents.compilerOptions.experimentalDecorators = true;
+			configContents.compilerOptions.removeComments = false;
+
+			configContents.compilerOptions.lib = [
+				...new Set([...(configContents.compilerOptions.lib || []), "es2017"]),
+			];
+			configContents.compilerOptions.plugins = [
+				...new Set([
+					...(configContents.compilerOptions.plugins || []),
+					{
+						transform:
+							"@nativescript/webpack/transformers/ns-transform-native-classes",
```

Comment:
This is only needed for plugins since projects get the transformer automatically through webpack.config.